lib/runners/config_generator.rb

```diff
@@ -8,7 +8,7 @@ def generate(tools: [])
     private
 
     def load_template(tools)
-      filename = __FILE__.sub(".rb", ".yml.erb")
+      filename = __FILE__.gsub(/.rb\Z/, ".yml.erb")
```

Comment:
[nitpick] I believe `lib/runners/config_generator.rb` will never be renamed to `lib/runners/config_generator!rb`, but it's just in case.

```suggestion
      filename = __FILE__.gsub(/\.rb\Z/, ".yml.erb")
```

Reply:
Same https://github.com/sider/runners/pull/2197#discussion_r596634338